pkg/webui/console/store/middleware/logics/events.js

```diff
@@ -177,16 +171,14 @@ const createEventsConnectLogics = (reducerName, entityName, onEventsStart) => {
       },
       process: async ({ action }, dispatch, done) => {
         if (action.type === START_EVENTS_FAILURE) {
-          if (action?.error?.message === 'timeout') {
-            // Set the connection status to `checking` to trigger connection checks
-            // and detect possible offline state.
-            dispatch(setStatusChecking())
+          // Set the connection status to `checking` to trigger connection checks
```

Comment:
I do not understand why this logic was guarded by the timeout error - Why does the error matter ? If the server instantly rejects the request, there won't be a timeout, but you should still retry at some point.